raiden/ui/cli.py

```diff
@@ -152,8 +145,8 @@ def options(func):
         option("--console", help="Start the interactive raiden console", is_flag=True),
         option(
             "--transport",
-            help="Transport system to use. UDP is not recommended",
-            type=click.Choice(["udp", "matrix"]),
+            help="Transport system to use.",
```

Comment:
As long as we only have matrix we could probably enable `hidden=True` on this option.